command/sub8_alarm/alarm_handlers/kill.py

```diff
@@ -8,13 +9,23 @@ class Kill(HandlerBase):
     def __init__(self):
         self._killed = False
         self._last_mission_killed = False
+        self.bagging_server = rospy.ServiceProxy('/online_bagger/dump', BaggerCommands)
 
     def raised(self, alarm):
         self._killed = True
+        self.bagger_dump()
 
     def cleared(self, alarm):
         self._killed = False
 
+    def bagger_dump(self):
+        ''' Dump online_bagger'''
+
+        try:
+            bag_status = self.bagging_server(bag_name='kill_bag', bag_time=0)
```

Comment:
By passing bag_time=0, won't we be bagging everything in the cache? Not sure how I feel about this as we may want a larger queue later down the line, but not want to dump the whole thing for a kill.